test/PublishCommand.js

```diff
@@ -907,6 +944,84 @@ describe("PublishCommand", () => {
         }
       }));
     });
+
+    /** =========================================================================
+    * INDEPENDENT - CD VERSION - PRERELEASE
+    * ======================================================================= */
+
+    it("should bump to prerelease versions with --cd-version=prerelease --prerelease-id=foo", (done) => {
```

Comment:
`prepatch` will make it  `1.0.1-beta.1` (actually, that's the difference between `prepatch` and `prerelease`)